doc/man3/DH_set_method.pod

```diff
@@ -31,7 +31,10 @@ Initially, the default DH_METHOD is the OpenSSL internal implementation, as
 returned by DH_OpenSSL().
 
 DH_set_default_method() makes B<meth> the default method for all DH
-structures created later. B<NB>: This is true only whilst no ENGINE has been set
+structures created later.
+This function is not thread-safe and should not be called at the same time
+as other OpenSSL functions.
+B<NB>: This is true only whilst no ENGINE has been set
 as a default for DH, so this function is no longer recommended.
```

Comment:
This last sentence sounds like it follows on from the previous one about thread safety